pkg/config/remote/client_test.go

```diff
@@ -17,8 +17,8 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/config"
 	rdata "github.com/DataDog/datadog-agent/pkg/config/remote/data"
 	"github.com/DataDog/datadog-agent/pkg/config/remote/meta"
-	"github.com/DataDog/datadog-agent/pkg/proto/pbgo"
-	"github.com/DataDog/datadog-agent/pkg/trace/pb"
+	pbgocore "github.com/DataDog/datadog-agent/pkg/proto/pbgo/core"
```

Comment:
Do you think it would make sense to prefix leaf package names with `pb` or `pbgo`, so that they don't need to be renamed every time we import them?